src/css/components/_volume.scss

```
@@ -144,6 +166,17 @@
   }
 }
 
+// Update placement of circle icon when using SVG icons
+.vjs-slider-horizontal .vjs-volume-level .vjs-svg-icon {
+  top: -0.2em;
+  right: -0.3em;
+  line-height: 0.35em;
```

```suggestion
  line-height: 0.15em;
```
And this seems to work on both better too

---

> Your last changes still didn't look right on my machine. Do my suggestions look right to you? Interesting and frustrating if we're getting different results.

The changes you suggested do look right, but the line height adjustment did not appear to make much of a difference in the progress slider. However, changing the smaller line-height in the volume slider did seem to make a bit of a difference. The sliders look good to me in both Chinese and non-Chinese in Chrome here: https://deploy-preview-8260--videojs-preview.netlify.app/sandbox/svg-icons-enabled.html